gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ServerInfo.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway.ha.clustermonitor;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static java.util.Objects.requireNonNull;
+
+// based on https://github.com/trinodb/trino/blob/f83854687229df1ae3325249e7bcd29f03d0943c/client/trino-client/src/main/java/io/trino/client/ServerInfo.java
```

Comment:
How about a tag version instead of a commit/blob just in case ServerInfo changes in future.

Reply:
Note that using commit hash is totally fine.